From 795302be2c1dae257c7d217258e378269323ee4b Mon Sep 17 00:00:00 2001
From: Igor Murzov <e-mail@date.by>
Date: Sun, 31 Mar 2013 11:30:11 +0400
Subject: [PATCH] Add audio/x-opus+ogg mime-type

--- shared-mime-info-1.2.orig/freedesktop.org.xml.in
+++ shared-mime-info-1.2/freedesktop.org.xml.in
@@ -2716,6 +2716,7 @@ command to generate the output files.
     </magic>
     <glob pattern="*.oga"/>
     <glob pattern="*.ogg"/>
+    <glob pattern="*.opus"/>
     <alias type="audio/x-ogg"/>
   </mime-type>
   <mime-type type="video/ogg">
@@ -2756,6 +2757,16 @@ command to generate the output files.
     <glob pattern="*.oga"/>
     <glob pattern="*.ogg"/>
   </mime-type>
+  <mime-type type="audio/x-opus+ogg">
+    <_comment>Opus audio</_comment>
+    <sub-class-of type="audio/ogg"/>
+    <magic priority="80">
+      <match type="string" value="OggS" offset="0">
+        <match type="string" value="OpusHead" offset="28"/>
+      </match>
+    </magic>
+    <glob pattern="*.opus"/>
+  </mime-type>
   <mime-type type="audio/x-speex+ogg">
     <_comment>Ogg Speex audio</_comment>
     <sub-class-of type="audio/ogg"/>

